packages/driver/cypress/e2e/cypress/proxy-logging.cy.ts

```diff
@@ -95,8 +96,7 @@ describe('Proxy Logging', () => {
 
             done()
           } catch (err) {
-            // eslint-disable-next-line no-console
-            console.log('assertion error, retrying', err)
+            done(new Error(err))
```

Comment:
This just gives us better error messaging. Without it, the error we'll see will be about `done` having never been called